roles/user-management/manage-users/tasks/create_user.yml

```diff
@@ -10,7 +10,7 @@
       name: "{{ item.user_name | trim }}"
       mail: "{{ item.email | default('') }}"
 # The addition of expiration date is a request that will be submitted upstream
-#      expiration_date: "{{ item.expiration_date | default('') }}"
+#      krbpasswordexpiration: "{{ item.expiration_date | default('') }}"
```

Comment:
This should be `krbprincipalexpiration`, or just leave it at the old value for now.